README.md

```diff
@@ -42,12 +42,14 @@ const initialState = { count: 0 };
 const { GlobalStateProvider, useGlobalState } = createGlobalState(initialState);
 
 const Counter = () => {
-  const [value, update] = useGlobalState('count');
+  const [count, setCount] = useGlobalState('count');
   return (
     <div>
-      <span>Counter: {value}</span>
-      <button onClick={() => update(v => v + 1)}>+1</button>
-      <button onClick={() => update(v => v - 1)}>-1</button>
+      <span>Counter: {count}</span>
+      {/* update state by passing callback function */}
+      <button onClick={() => setCount(v => v + 1)}>+1</button>
+      {/* update state by passing new value */}
+      <button onClick={() => setCount(v - 1)}>-1</button>
```

Comment:
```suggestion
      <button onClick={() => setCount(count - 1)}>-1</button>
```

Reply:
@piggyman007 Would you commit this change?

Reply:
yes, it is good to provide many examples

`setCount(count - 1)` this seems like the native hook and it is easy to understand and adopt

Reply:
I agree. I mean I just need you to click "commit suggestion" and "commit changes" above.

Reply:
ok

Reply:
Or, just fix L52 if you are not very familiar with GitHub. Whichever is fine.

Reply:
@dai-shi 

I don't understand. What change should I do?

Reply:
What I meant in https://github.com/dai-shi/react-hooks-global-state/pull/23#discussion_r314777406 is `setCount(v - 1)` should be `setCount(count - 1)`.
Otherwise, it doesn't work, does it?

Reply:
got it, thanks